### PR TITLE
VFB-172 error fix parcelId function

### PR DESCRIPTION
**Pull request by mtharkins**
## What's changed
Error handling added to prevent error when loading parcels page

## Checklist
- [x] The ticket is up-to-date - Please document any deviations from the original approach if there is any.
- [x] I have documented the testing steps for QA
- [x] I have self-reviewed this PR
- [x] Make sure you've verified it works via `npm run dev`
- [x] Make sure you've verified it works via `npm run build` and `npm run start`
- [x] Make sure you've fixed all linting problems with `npm run lint_fix`
- [x] Make sure you've tested via `npm run test:e2e`

If you have made any changes to the database...
  - [x] The migration files are up-to-date with my final set up (`npx supabase db diff -f <name_of_migration>` should create nothing at this point)
  - [x] I have updated the typescript definitions for the database with `db:local:generate_types`
  - [x] I have modified the seed in `supabase/seed/seed.mts` if appropriate
  - [x] If I have modified the seed, I have also generated the seed with `npm run db:generate_seed` 
  - [x] With my final set up, I can run `npm run dev:reset_supabase` without any errors.
  - Does this require resetting the deployed database? - NO (remove as appropriate)

### CODE REVIEW

File: src/app/parcels/form/ParcelForm.tsx
Comment (rickaw) [inline src/app/parcels/form/ParcelForm.tsx:1]: is this file intended?

File: supabase/seed.sql
Comment (rickaw) [inline supabase/seed.sql:1]: is this intended?

**Review by rickaw — CHANGES_REQUESTED** [review]

File: src/app/parcels/form/ParcelForm.tsx
Comment (mtharkins) [reply src/app/parcels/form/ParcelForm.tsx:1]: When I ran lint_fix it suggested that these were needed, I can remove though if you don't think it's necessary

File: supabase/seed.sql
Comment (mtharkins) [reply supabase/seed.sql:1]: This was probably from when I was trying to resolve the errors I was getting and ran some database commands. What is the best way to exclude these changes.

File: src/app/parcels/form/ParcelForm.tsx
Comment (rickaw) [reply src/app/parcels/form/ParcelForm.tsx:1]: let's remove those changes for now, we can fix this separately :)

File: supabase/seed.sql
Comment (rickaw) [reply supabase/seed.sql:1]: Probably the easiest thing to do would be to
- go to the base or your branch to copy the content of the original seed.sql
- go back to the latest commit on your branch and paste the original content into seed.sql, and commit

**Comment by rickaw** [comment]
Sorry, just realised this change is covered in #220 (which I made! but forgot this change was part of it!) So happy to close this PR! I should've clocked it sooner!
